libs/template/cli/src/commands/create.ts

```diff
@@ -239,13 +231,22 @@ interface CreateAppConfig extends Required<CreateAppOptions> {
 export interface CreateAppOptions {
   name?: string;
   preset?: OryxPreset;
+  options?: OryxOption[];
 }
 
 export enum OryxPreset {
```

Comment:
Probably we can remove it as long as we are not using it yet.

Reply:
we can, up to you :)